editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -381,12 +385,13 @@ void TileSetAtlasSourceEditor::AtlasTileProxyObject::_get_property_list(List<Pro
 	}
 
 	// ID and size related properties.
+	p_list->push_back(PropertyInfo(Variant::NIL, TTR("Tile"), PROPERTY_HINT_NONE, String(), PROPERTY_USAGE_CATEGORY));
```

Comment:
Maybe it should be changed to Alternative Tile when an alternative tile is selected.

Reply:
I changed this so it's "Base Tile" or "Alternative Tile" when only one tile is selected, and it's "Tiles" when multiple tiles are selected.